keyboards/dumbpad/v1x_oled/keymaps/default/keymap.c

```diff
@@ -0,0 +1,207 @@
+
+#include <stdio.h>
+char wpm_str[10];
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+        /*
+            BASE LAYER
+    /-----------------------------------------------------`
+    |             |    7    |    8    |    9    |  Bkspc  |
+    |             |---------|---------|---------|---------|
+    |             |    4    |    5    |    6    |   Esc   |
+    |             |---------|---------|---------|---------|
+    |             |    1    |    2    |    3    |   Tab   |
+    |-------------|---------|---------|---------|---------|
+    | Play Pause  |  TT(1)  |    0    |    .    |  Enter  |
+    \-----------------------------------------------------'
+    */
+    [0] = LAYOUT(
+                    KC_7,      KC_8,    KC_9,             KC_BSPC,
+                    KC_4,      KC_5,    KC_6,             KC_ESC,
+                    KC_1,      KC_2,    KC_3,             KC_TAB,
+        KC_MUTE,    TT(1),     KC_0,    LSFT_T(KC_DOT),   KC_ENTER
+    ),
+    /*
+            SUB LAYER
+    /-----------------------------------------------------`
+    |             |         |         |         |  Reset  |
+    |             |---------|---------|---------|---------|
+    |             |         |         |         |    +    |
+    |             |---------|---------|---------|---------|
+    |             |         |         |         |    -    |
+    |-------------|---------|---------|---------|---------|
+    |    LOCK     |         |         |         |    =    |
+    \-----------------------------------------------------'
+    */
+    [1] = LAYOUT(
+                    _______,     _______,     _______,      RESET,
+                    _______,     _______,     _______,      KC_KP_PLUS,
+                    _______,     _______,     _______,      KC_KP_MINUS,
+        KC_LOCK,    _______,     _______,     _______,      KC_EQL
+    ),
+
+    [2] = LAYOUT(
+                    _______,     _______,     _______,      _______,
+                    _______,     _______,     _______,      _______,
+                    _______,     _______,     _______,      _______,
+        _______,    _______,     _______,     _______,      _______
+    ),
+
+    [3] = LAYOUT(
+                    _______,     _______,     _______,      _______,
+                    _______,     _______,     _______,      _______,
+                    _______,     _______,     _______,      _______,
+        _______,    _______,     _______,     _______,      _______
+    )
+};
+
+#ifdef ENCODER_ENABLE
+bool encoder_update_user(uint8_t index, bool clockwise) {
+    // Right encoder
+    if (index == 0) {
+        if (clockwise) {
+            tap_code(KC_VOLU);
+        } else {
+            tap_code(KC_VOLD);
+        }
+    // Left encoder
+    } else if (index == 1) {
+        if (clockwise) {
+            tap_code(KC_MNXT);
+        } else {
+            tap_code(KC_MPRV);
+        }
+    }
+    return true;
+}
+#endif
+
+#ifdef OLED_ENABLE
```

Comment:
ALso, this should probably be moved to the `v1x_oled.c` file, like with the encoder.

Reply:
Moved OLED code out of keymap files in https://github.com/qmk/qmk_firmware/pull/17058/commits/165318675c78d8f4697645e237d2aff5672aa12d